devices/lidl.js

```diff
@@ -270,6 +270,14 @@ module.exports = [
         extend: extend.light_onoff_brightness({disableEffect: true}),
         meta: {turnsOffAtBrightness1: false},
     },
+    {
+        fingerprint: [{modelID: 'TS0501A', manufacturerName: '_TZ3000_nosnx7im'}],
```

Comment:
`_TZ3000_nosnx7im` is added as `HG06462A` (below), but the device was not recognised (for me) and is listed as `HG06463B` on the box.